app/assets/stylesheets/base/boxes.scss

```diff
@@ -0,0 +1,40 @@
+@import '../vars/variables';
+
+@mixin box($this: $this, $color-1: $smoke-90, $color-2: $smoke-100) {
+  border-color: $color-2;
+
+  &#{$this}--filled {
+    background-color: $color-1;
+  }
+
+  @for $i from 1 through 3 {
+    &#{$this + '--level-' + $i} {
+      box-shadow: #{pow(2, $i) + px} #{pow(2, $i) + px} 0 $color-2;
+    }
+  }
+}
+
+.crayons-box {
```

Comment:
will all our design system-related CSS be prefixed by `crayons-`?

Reply:
Components - yes.

Reason: we have tons of existing CSS in codebase - this will:
- help making code unique
- help avoid overlapping
- make it more readable in the markup so you can spot the new/correct approach right away without digging in